.changeset/soft-lions-warn.md

```diff
@@ -1,5 +0,0 @@
----
-"@hashicorp/design-system-components": minor
----
-
-Initial commit to open PR (to be removed later)
```

Comment:
@didoo I think this snuck in and should be removed before we do the release

Reply:
yeah, I noticed too: https://github.com/hashicorp/design-system/pull/2070